page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/allure/ErrorHandler.java

```diff
@@ -23,11 +23,9 @@ public static void attachScreenshot() {
         }
     }
 
-    public static void attachError(String title, Throwable throwable) {
+    public static void attachError(Throwable e) {
```

Comment:
we need to supply a custom error title. so revert back signature. or create two methods